Samples/CSharp/FieldsExamples.cs

```diff
@@ -31,20 +34,20 @@ internal class FieldsExamples
          */
 
         // Allowed by both .NET Guidelines and StyleCop, and are properly PascalCased
-        public        const    int PascalCasePublicConstField          = 1;
+        public const int PascalCasePublicConstField = 1;
```

Comment:
This comment applies to the entirety of the file.

I want to ensure you intentionally un-aligned the (previously) vertically aligned text in this file.  
I ask because this is stylistic, and I had preferred the vertical alignment (made it easier for me to visually review the permutations / combinations).

If this was unintentional, please revert.
Otherwise, as this is stylistic, you may do with this as you wish.

Reply:
I'm not sure what caused this. I think something auto formatted the code when I was messing with the file header. I'm not too bothered by it, so I'll revert when I get a chance tomorrow.

Reply:
I think this happened because I'm using the [Trailing Whitespace Visualizer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer) VS extension which auto formats files on save. I've never noticed it before because I don't normally layout code in a grid fashion. I used notepad to make the changes for now.